frontend/ui/wikipedia.lua

```diff
@@ -747,14 +758,19 @@ time, abbr, sup {
 </html>
 ]], page_cleaned, page_htmltitle, lang:upper(), saved_on, see_online_version, html))
 
+    -- Force a GC to free the memory we used till now
+    collectgarbage()
+    collectgarbage()
```

Comment:
Probably add a comment about the doing it twice hack too. (Assuming it's not a typo.)